packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -48,8 +48,12 @@ class CreateCommand extends CreateBase {
         flutterProjectTypeToString(FlutterProjectType.package): 'Generate a shareable Flutter project containing modular '
             'Dart code.',
         flutterProjectTypeToString(FlutterProjectType.plugin): 'Generate a shareable Flutter project containing an API '
-            'in Dart code with a platform-specific implementation for Android, for iOS code, or '
-            'for both.',
+            'in Dart code with a platform-specific implementation through method channels for Android, iOS, '
+            'Linux, MacOS, Windows, web, or for any combination of these.',
```

Comment:
I can go and find all places where I updated comments and did cleanups that are not strictly FFI plugins and make a separate PR if you'd prefer. (It would be a PR with a ton of small fixes without a lot of context though.) Let me know what you prefer.

Reply:
This should be fine